joomla/joomla-cms
# Added the generator meta tag option to Configuration Manager.

Pull Request for a new feature.

### Summary of Changes
No administrators can choose what data should be display in the generator meta tag in html code and in feeds. 
- Default
Gives default value: Joomla! - Open Source Content Management.
- Domain
Set your domain as value. 
- Custom text
Allow to set an own text as value. 
- Hide
Allow to hide the generator meta tag. 

### Testing Instructions
Use html code preview (Ctrl + U in Firefox) to observe changes. 
To preview the code of feeds, click on links to feeds. 
Remember to refresh website and code preview after each change. 

### Expected result
Different values in the generator meta tag or no generator meta tag. 

### Actual result
Everywhere is displayed the same value "Joomla! - Open Source Content Management". The value can't be changed without a plugin, which not always work properly. 

### Documentation Changes Required
Information about new features in Configuration Manager.
